.pipelines/templates/template-job-deploy-azure-env-tag.yml

```diff
@@ -36,7 +36,7 @@ jobs:
               azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
           - script: |
               set -e
-              trap 'set +e; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af ; rm -rf ~/.docker/config.json' EXIT
+              trap 'set +e; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af ; rm -rf ~/.docker/config.json; rm -rf /run/user/1000/containers/auth.json' EXIT
```

Comment:
Does $USERNAME work instead of 1000?

Reply:
no, $USERNAME is `cloud-user` which is not the same path.

Reply:
oh, it's not inside the container. Gotcha.

Reply:
but may want to do a `rm -rf /run/user/$(id $USERNAME)/containers/auth.json`

Reply:
Updated.